ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+   |                       |4.2.4     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |start                  |[RFC8984],|DTSTART     |Section 2.3.17     |
+   |                       |Section   |            |                   |
+   |                       |4.7.2     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |timeZone               |[RFC8984],|TZID        |Section 2.3.17,    |
+   |                       |Section   |(parameter) |Section 2.3.18     |
+   |                       |4.7.1     |of DTSTART  |                   |
+   |                       |          |and DUE     |                   |
+   +-----------------------+----------+------------+-------------------+
+   |timeZones              |[RFC8984],|VTIMEZONE   |TimeZone object    |
+   |                       |Section   |(component) |(Section 3.7)      |
+   |                       |4.7.2     |of iCalendar|                   |
+   |                       |          |object      |                   |
+   +-----------------------+----------+------------+-------------------+
+   |title                  |[RFC8984],|SUMMARY     |Section 2.3.45     |
+   |                       |Section   |            |                   |
+   |                       |4.2.1     |            |                   |
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 86]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   +-----------------------+----------+------------+-------------------+
+   |uid                    |[RFC8984],|UID         |Section 2.3.55     |
+   |                       |Section   |            |                   |
+   |                       |4.1.2     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |updated                |[RFC8984],|DTSTAMP, and|Section 2.3.16,    |
+   |                       |Section   |LAST-       |Section 2.3.24     |
+   |                       |4.1.6     |MODIFIED    |                   |
+   +-----------------------+----------+------------+-------------------+
+   |useDefaultAlerts       |[RFC8984],|TBD         |                   |
+   |                       |Section   |            |                   |
+   |                       |4.5.1     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+   |virtualLocations       |[RFC8984],|CONFERENCE  |Section 2.3.12     |
+   |                       |Section   |            |                   |
+   |                       |4.2.6     |            |                   |
+   +-----------------------+----------+------------+-------------------+
+
+             Table 30: Properties of the Event and Task object
+
+   Remarks:
+
+   *  The descriptionContentType property value determines if to convert
```

Comment:
"if to convert" feels weird.

Maybe...
"The descriptionContentType property value determines if the description property should be converted to the DESCRIPTION or STYLED-DESCRIPTION property."